syntax = "proto3";

package computationpacket;

message ComputationPacket {
  string region_name = 1;
  uint32 size = 2;
}